src/components/Header.tsx

```diff
@@ -10,7 +11,8 @@ import profileIcon from '../assets/icons/profile.svg';
 
 const Header = (): ReactElement => {
   const navigate = useNavigate();
-  const [isLoggedIn, setIsLoggedIn] = useState(false);
+  const isAuthenticated = useIsAuthenticated;
```

Comment:
does it not work with 
```suggestion
  const isAuthenticated = useIsAuthenticated();
```
and below calling `isAuthenticated` instead of `isAuthenticated()`? hooks should be generally called like that

Reply:
truuuew... it did not work before when i tred it but now it does 🤠👍